assets/src/edit-story/components/library/panes/media/mediaPane.js

```diff
@@ -79,7 +80,9 @@ const FILTERS = [
   { filter: 'video', name: __('Video', 'web-stories') },
 ];
 
-const DEFAULT_WIDTH = 150;
+// By default, the element should be 50% of the page.
+const DEFAULT_ELEMENT_WIDTH = PAGE_WIDTH / 2;
```

Comment:
Should the default width perhaps come from the type definition?

Reply:
I thought about that as well, but figured this is still a good step to start with. If we need more details, the next step is definitely push this to element definition.